source/architecture/single_page.md

```diff
@@ -0,0 +1,42 @@
+# Build a single page site
+
+You can create a technical documentation site that contains all of its content in a single page.
+
+This is suitable for documentation sites that have do not have a lot of content, and do not need to have topics split into individual pages.
+
+## Organise content files 
+
+1. Create [link] the content files for your documentation site.
+
+1. Amend [link] your content files to ready them for your documentation site.
+
+1. Save your content files inside your documentation repo in your desired hierarchy.
+
+## Build your documentation site's structure
+
+1. Go to the __source__ folder in your documentation repo.
+
+1. Select the `index.html.md.erb` file.
+
+1. Amend the `index.html.md.erb` file, adding a `<%= partial` line for each content file that makes up your overall documentation site. 
```

Comment:
Partial files aren't mandatory, so it might be better to talk about the different options